packages/template-ui/src/views/Section.vue

```diff
@@ -38,7 +38,8 @@ export default {
       if (!this.isSimpleBundle) {
         return false;
       }
-      return this.node.topic_children_count === 0;
+      const hasChildTopics = this.sectionNodes.some((n) => n.kind === 'topic');
```

Comment:
This is not ideal. Please add a FIXME comment line:
```suggestion
      // FIXME: Use API to query the amount of subtopics:
      const hasChildTopics = this.sectionNodes.some((n) => n.kind === 'topic');
```